catalog-info.yaml

```diff
@@ -0,0 +1,24 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'Credentials Themes'
+  description: 'A required package for the Credentials IDA. It provides the frontend layouts and visual styling used to render Program Certificates, Learner Records, and Program Records.'
+  links:
+    - url: 'https://github.com/openedx/credentials-themes/blob/master/README.rst'
+      title: 'Documentation'
+      icon: 'Article'
+  annotations:
+    # (Optional) Annotation keys and values can be whatever you want.
+    # We use it in Open edX repos to have a comma-separated list of GitHub user
+    # names that might be interested in changes to the architecture of this
+    # component.
+    openedx.org/arch-interest-groups: ""
+    # This can be multiple comma-separated projects.
+    openedx.org/add-to-projects: "openedx:23"
+spec:
+  type: 'service'
```

Comment:
I think this should be `library` rather than service.

Reply:
Makes sense! Making the change now